packages/util/src/use-resize-observer.ts

```diff
@@ -1,12 +1,12 @@
 import { canUseDOM } from 'exenv'
-import { RefObject, useEffect, useState } from 'react'
+import { MutableRefObject, RefObject, useEffect, useState } from 'react'
 import ResizeObserver from 'resize-observer-polyfill'
 
 import { combineFns } from './combine-fns'
 import { debounce } from './debounce'
 
 export function useResizeObserver(
-  ref: RefObject<HTMLElement>,
+  ref: RefObject<HTMLElement> | MutableRefObject<HTMLElement | undefined>,
```

Comment:
add MutableRefObject since that's the type returned from forwardRef.  Needed in this case.  And probably a common case.